lib/util.js

```diff
@@ -355,3 +355,55 @@ exports.stripHTML = (html) => {
     .replace(/<.*?>/gi, '')
     .trim();
 };
+
+/* Match begin and end braces of input JSON, return only json 
+ * it works with braced jsons only!
+ * @param {String} mixedJson
+ * @return {String}
+*/
+exports.cutAfterJSON = function(mixedJson) {
+  let counter = 0, pos = 0, isString, prevChar, open, close;
+  if (mixedJson[0] === '[') {
+    open = '[';
+    close = ']';
+  }
+  if (mixedJson[0] === '{') {
+    open = '{';
+    close = '}';
+  }
+  if (!open) {
+    throw new Error("can't cut unsupported JSON (need to begin with [ or { ) but got " + mixedJson[0]);
```

Comment:
try to stick with always using `'`

Reply:
Would be nice to have eslint config, usually we not avoid double quotes if there is single quote inside.

Reply:
also love eslint
university is keeping me pretty busy thou...